proposal/ycchan5566/README.md

```diff
@@ -0,0 +1,56 @@
+# Proposal for Physics Engine
+
+## Basic information
+
+Implement a c++ base physics engine and use python to write a easy physical simulator.
+
+The GitHub repository:
+https://github.com/ycchan5566/Physics-Engine
+
+## Problem to solve
+
+Today, the number of physics engine provided by python is still small, so I decide to develop a physics engine in order to enrich the physical library of python.
```

Comment:
I think you need to be more specific about what physical phenomena you want to simulate.  I guess you are not planning to simulate everything about physics.